packages/salesforcedx-vscode-core/src/context/workspaceContext.ts

```diff
@@ -59,3 +63,16 @@ export class WorkspaceContext {
     return WorkspaceContextUtil.getInstance().alias;
   }
 }
+
+async function setIsScratchOrg() {
+  const username = await ConfigUtil.getUsername();
+  if (!username) {
+    return;
+  }
+  const isScratch = await OrgAuthInfo.isAScratchOrg(String(username));
```

Comment:
Two comments:
1. `username` was already check for being undefined in line 69.  Because of this, the `String()` on line 72 isn't needed.
2. nit-pick: my preference is to not abbreviate variable names, and pref it named, `isScratchOrg`